about.html

```diff
@@ -0,0 +1,162 @@
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Capstone Project</title>
+  </head>
+  <body>
+    <header>
+      <div class="nav-bar1">
+        
+          <a href="facebook.com/promzzyokechukwu"><i class="fa-brands fa-facebook-f"></i></a>
+          <a href="facebook.com/promzzyokechukwu"><i class="fa-brands fa-twitter"></i></a>
+          <a href="#">Logout</a>
+          <a href="https://promzzykoncepts.github.io/My-Portfolio/">My Page</a>
+          <a href="#">English</a>
+        </div>
+      
+      <nav class="nav-bar">
+
+        
+      <ul class="nav-menu">
+        <li class="nav-item logo">
+          
+          <a href="#"><img class="logo-img" src="images/R.png" alt="logo"/> 
+          <h3>REMOTE GLOBAL  SUMMIT<br>2022</h3></a>
+          
+        </li>
+          <li class="nav-item">
+              <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-item">
+              <a href="" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+              <a href="" class="nav-link">Program</a>
+          </li>
+          <li class="nav-item">
+              <a href="" class="nav-link">Join</a>
+          </li>
+             <li class="nav-item labelT">
+            <a href="" class="nav-link">WFH Remoters..</a>
+        </li>
+      </ul>
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+          
+      </div>
+      </nav>
+  </header>
+
+    <section class="mainS"></section>
+           <div class="main-header">
+            <p class="hello">"Hello! Am a WFH Work from Home"</p>
+                   <h1 class="header-text">
+                        REMOTE DEV'S ANNUAL GLOBAL &nbsp;SUMMIT 2022 
+          </h1>
+          <p class="desc"> The after month of covid 19 saw an over whelming growth of the demand for remote workers
+            Previously a very slow sector experienced a sharp demand as companies had also come to appreciate thie benefits associtaed. the initially 
+            unknown event now gathers all works of life with a turn up of thousands each year. For the first Time, its in AFRICA , Kampala, Uganda in E.Africa this OCTOBER</p>
+             <p class="desc2">Please contact @www.MunyonyoCommonwealth esort.ug, Along Lake Victoria Shores, @www.TripeAdvisors.com, www.Eaafricansafari.com</p>
+             
+             </div>
```

Comment:
- [ ] In the about page for the mobile version I think here you should fix the heading alignmenet as I can see there is more space vetween the header and the menu so kindly fix that ✅👍🏻

- Your design 
![image](https://user-images.githubusercontent.com/58553711/183260898-98e025c9-4061-4f22-83d3-d965efa4e0fe.png)

- [ ] In the desktop version as I can see from my side the height for the first section is too big so kindly work on that to fix it to be identical to the original design 👍🏻✅✅✅ so in the first section I just can see the header 😅

- Your design 
![image](https://user-images.githubusercontent.com/58553711/183261039-82221bb7-4840-4769-892f-5e8e4ab8e1ea.png)

- Original design
![image](https://user-images.githubusercontent.com/58553711/183261120-bc6df54f-1d65-4b0e-9e48-c6969cf4ebd9.png)